pgjdbc/src/main/java/org/postgresql/jdbc/PgPreparedStatement.java

```diff
@@ -1498,6 +1504,11 @@ private void setTimestamp(@Positive int i, OffsetDateTime offsetDateTime)
     bindString(i, getTimestampUtils().toString(offsetDateTime), oid);
   }
 
+  private void setTimestamp(@Positive int i, Instant instant)
+      throws SQLException {
+    setTimestamp(i, instant.atOffset(ZoneOffset.UTC));
```

Comment:
There is `Timestamp.from(instant)` ... to turn instant into java.sql.Timestamp.

Reply:
I would say java.sql.Timestamp is not reliable when it comes to timezones, so sending offsetdatetime sounds better to me than java.sql..

Reply:
ok cool.